packages/client-core/src/user/components/UserMenu/menus/AvatarSelectMenu.tsx

```diff
@@ -63,6 +65,8 @@ const AvatarMenu = () => {
   const avatarLoading = useHookstate(false)
   const isUserReady = useHookstate(getMutableState(LocalAvatarState).avatarReady)
 
+  const createAvatarEnabled = FeatureFlagsState.useEnabled(FeatureFlags.Client.Menu.CreateAvatar)
```

Comment:
This line is causing the check errors task to fail. I think this needs to be updated to FeatureFlags.Client.Admin.CreateAvatar